examples/token/nep5/nep5.go

```diff
@@ -93,3 +93,19 @@ func IsUsableAddress(addr []byte) bool {
 
 	return false
 }
+
+// Mint initial supply of tokens.
+func (t Token) Mint(ctx storage.Context, to []byte) bool {
+	if !IsUsableAddress(t.Owner) {
+		return false
+	}
+	minted := storage.Get(ctx, []byte("minted")).(bool)
+	if minted {
+		return false
+	}
+
+	storage.Put(ctx, to, t.TotalSupply)
+	storage.Put(ctx, []byte("minted"), true)
```

Comment:
The only difference from your code is in this `true` value in order to prevent double-initialisation. Tested this contract, actual behaviour now:
1. Mint token to account `AKkkumHbBipZ46UMZJoFynJMXzSRnBvKcs`
2. All available amount of token is on `AKkkumHbBipZ46UMZJoFynJMXzSRnBvKcs` account (checked with `balansOf`)
3. There's no token on `AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y` account (checked with `balanceOf`)
4. Can not mint one more time (`mint` returned false to stack)

Reply:
So it was almost correct, nice.